### PR TITLE
Solve and enforce 9 pylint messages

### PR DESCRIPTION
**Pull request by berland**
**Issue**
Adds to #3309 

**Approach**
Do as pylint suggests

## Pre review checklist

- [x] Added appropriate release note label
- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [ ] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).

Adding labels helps the maintainers when writing release notes. This is the [list of release note labels](https://github.com/equinor/ert/labels?q=release-notes).

### CODE REVIEW

**Comment by berland** [comment]
Will squash after review.

File: src/ert/_c_wrappers/enkf/key_manager.py [hunk 1]
```diff
@@ -61,10 +56,9 @@ def summaryKeysWithObservations(self) -> List[str]:
 
     def genKwKeys(self) -> List[str]:
         if self.__gen_kw_keys is None:
-            gen_kw_keys = self.ensembleConfig().getKeylistFromImplType(
-                ErtImplType.GEN_KW
+            gen_kw_keys = list(
+                self.ensembleConfig().getKeylistFromImplType(ErtImplType.GEN_KW)
```
Comment (xjules) [inline src/ert/_c_wrappers/enkf/key_manager.py:60]: Good that you fixed all `getKeyList*` function! :+1:

File: src/ert/_c_wrappers/job_queue/ext_job.py [hunk 2]
```diff
@@ -231,7 +231,7 @@ def __eq__(self, other):
         if self.name() != other.name():
             return False
 
-        if [x for x in self.get_arglist()] != [x for x in other.get_arglist()]:
+        if list(self.get_arglist()) != list(other.get_arglist()):
```
Comment (xjules) [inline src/ert/_c_wrappers/job_queue/ext_job.py:234]: Doesn't `get_arglist()` already return `list`?

Comment (berland) [reply src/ert/_c_wrappers/job_queue/ext_job.py:234]: It does indeed! Fixed.

**Comment by hnformentin** [comment]
I did not see that @xjules was on this review before I assigned myself. Will let it with him!

**Comment by berland** [comment]
> I did not see that @xjules was on this review before I assigned myself. Will let it with him!

If you have gone through everything and found nothing to pick on, feel free to approve even with xjules assigned!

**Review by xjules — APPROVED** [review]
Great job! I think it can go in now :)

**Pull request merged**